example/use_cases/apartment_application/apartment_application.go

```diff
@@ -0,0 +1,150 @@
+
+	fmt.Print("\n\nStep 1: Create new DIDs for entities\n\n")
+	fmt.Printf("Tenant: %s\n", string(*holderDIDKey))
+	fmt.Printf("Apartment: %s\n", string(*aptDIDKey))
+	fmt.Printf("Government: %s\n", string(*govtDIDKey))
+
+	/**
+		 Step 2: Government issuer issues credentials to holder claiming age. The government issuer then signs the verifiable credentials to holder claiming age
+	**/
+
+	knownIssuer := govtDIDKey
+	knownIssuanceDate := "2020-01-01T19:23:24Z"
+	knownSubject := map[string]interface{}{
+		"id":        string(*holderDIDKey),
+		"birthdate": "1975-01-01",
+	}
+
+	vcBuilder := credential.NewVerifiableCredentialBuilder()
+
+	vcBuilder.SetIssuer(string(*knownIssuer))
+	vcBuilder.SetIssuanceDate(knownIssuanceDate)
+	vcBuilder.SetCredentialSubject(knownSubject)
+
+	vc, _ := vcBuilder.Build()
+	assert.NoError(nil, vc.IsValid())
+
+	signedVCBytes, _ := signing.SignVerifiableCredentialJWT(*govtSigner, *vc)
+
+	fmt.Print("\n\nStep 2: Government issues Verifiable Credential new for tenant verifying birthdate and signs\n\n")
+	if dat, err := util.PrettyJSON(vc); err == nil {
+		fmt.Printf("Verifiable Credential:%s\n", string(dat))
+	}
+
+	/**
+		Step 3: Create presentation definition from the apartment to the holder which goes into a presentation request.
+		The apartment is saying "here tenant, here is my what information I am requesting from you"
+	**/
+
+	presentationDefinitionBuilder := exchange.NewPresentationDefinitionBuilder()
+
+	presentationDefinitionBuilder.SetInputDescriptors([]exchange.InputDescriptor{
+		{
+			ID:      "birthdate",
+			Purpose: "Age verification",
+			Format: &exchange.ClaimFormat{
+				JWTVC: &exchange.JWTType{Alg: []crypto.SignatureAlgorithm{crypto.EdDSA}},
+			},
+			Constraints: &exchange.Constraints{Fields: []exchange.Field{
+				{
+					Path: []string{"$.credentialSubject.birthdate"},
+					ID:   "birthdate",
+				},
+			}},
+		},
+	})
+
+	presentationDefinition, _ := presentationDefinitionBuilder.Build()
+	assert.NoError(nil, presentationDefinition.IsValid())
+
+	presentationRequestBytes, _ := exchange.BuildPresentationRequest(aptSigner, exchange.JWTRequest, *presentationDefinition, string(*holderDIDKey))
+
+	fmt.Print("\n\nStep 3: The apartment creates a presentation request that confirms which information is required from the tenant\n\n")
+	if dat, err := util.PrettyJSON(presentationDefinition); err == nil {
+		fmt.Printf("Presentation Definition that gets added to presentation request:%s\n", string(dat))
+	}
+
+	/**
+		Step 4: Tenant holder verifies the presentation request from the apt is valid and then constructs and signs a presentation submission
+	**/
+
+	verifiedPresentationDefinition, err := exchange.VerifyPresentationRequest(aptVerifier, exchange.JWTRequest, presentationRequestBytes)
+	assert.NoError(nil, verifiedPresentationDefinition.IsValid())
+
+	// TODO: Have the presentation claim's token format support signedVCBytes for the BuildPresentationSubmission function
```

Comment:
nit: for TODOs, generally advisable to link to an issue (or at least create one). I know I haven't been great about this either.